libs/librepcb/projecteditor/boardeditor/boardeditor.ui

```diff
@@ -109,6 +109,9 @@
     <addaction name="actionToolDrawPolygon"/>
    </widget>
    <widget class="QMenu" name="menuHelp">
+    <property name="enabled">
```

Comment:
I would change "visible" to false instead of "enabled", because IMO it looks strange to have a disabled menu item.